Automatic Validation ended with: 
> 2026-04-11T06:39:02.6219186Z ##[error] Manifest Error: Schema Error. [Schema validation failed. Error context: <root>[Installers][0][InstallerSha256] Description: Failed to match regex specified by 'pattern' constraint. Error context: <root>[Installers][0] Description: Failed to validate against schema associated with property name 'InstallerSha256'. Error context: <root>[Installers] Description: Failed to validate item #0 in array. Error context: <root>[Installers][1][InstallerSha256] Description: Failed to match regex specified by 'pattern' constraint. Error context: <root>[Installers][1] Description: Failed to validate against schema associated with property name 'InstallerSha256'. Error context: <root>[Installers] Description: Failed to validate item #1 in array. Error context: <root> Description: Failed to validate against schema associated with property name 'Installers'. ] File: 6646cf37c6573c7804aa39bb5fd56ced300b32e1bbd393fd262929f503f9f104 Manifest Error: Failed to process field. [InstallerSha256] Manifest Error: Failed to process field. [InstallerSha256] Manifest Error: Required field missing. [InstallerSha256] Manifest Error: Required field missing. [InstallerSha256]
2026-04-11T06:39:02.6220421Z ##[error] Manifest Error: Schema Error. [Schema validation failed. Error context: <root>[Installers][0][InstallerSha256] Description: Failed to match regex specified by 'pattern' constraint. Error context: <root>[Installers][0] Description: Failed to validate against schema associated with property name 'InstallerSha256'. Error context: <root>[Installers] Description: Failed to validate item #0 in array. Error context: <root>[Installers][1][InstallerSha256] Description: Failed to match regex specified by 'pattern' constraint. Error context: <root>[Installers][1] Description: Failed to validate against schema associated with property name 'InstallerSha256'. Error context: <root>[Installers] Description: Failed to validate item #1 in array. Error context: <root> Description: Failed to validate against schema associated with property name 'Installers'. ] File: 6646cf37c6573c7804aa39bb5fd56ced300b32e1bbd393fd262929f503f9f104 Manifest Error: Failed to process field. [InstallerSha256] Manifest Error: Failed to process field. [InstallerSha256] Manifest Error: Required field missing. [InstallerSha256] Manifest Error: Required field missing. [InstallerSha256]

(Automated response - build 1247.)

--- a/manifests/q/qr243vbi/NekoBox/5.10.36/qr243vbi.NekoBox.installer.yaml
+++ b/manifests/q/qr243vbi/NekoBox/5.10.36/qr243vbi.NekoBox.installer.yaml
@@ -22,9 +22,9 @@ InstallationMetadata:
 Installers:
 - Architecture: x64
   InstallerUrl: https://github.com/qr243vbi/nekobox/releases/download/5.10.36/nekobox-5.10.36-windows64-installer.exe
-  InstallerSha256: OldSha64
+  InstallerSha256: 4071cce063c49f34b72a4de0071c92bf92cc2d668cd2eaa5f76682cace46463c
 - Architecture: arm64
   InstallerUrl: https://github.com/qr243vbi/nekobox/releases/download/5.10.36/nekobox-5.10.36-windows-arm64-installer.exe
-  InstallerSha256: OldShaArm
+  InstallerSha256: b41fc9fd76731d1e8ed128300a04511fa7d93a5bb460fb2e737dded9b693413d
 ManifestType: installer
 ManifestVersion: 1.10.0

